<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="line_number_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_result">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_number_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="result_b">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Посчитать</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>211</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QRadioButton" name="minus_Rb">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="sin_Rb">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="sqrt_Rb">
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="multiply_Rb">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="summ_Rb">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="del_Rb">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="error_Lb">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>400</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_Lb">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>232</width>
     <height>69</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;результат&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="op2_Lb">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>111</width>
     <height>69</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;элемент 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="op1_Lb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;        элемент 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>160</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="sin_Rb_rad">
      <property name="text">
       <string>радианы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sin_Rb_grad">
      <property name="text">
       <string>градусы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
